### DISCUSSION TITLE
container/image

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#144] [General]
# container/image

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/image.html

**Comment by dentonliu** [comment]
会发现是这一个 rootfs 目录  应为  会发现这是一个 rootfs 目录

**Reply by isno** [reply]
这一篇我还没有完稿，不过谢谢提醒。

**Comment by echo1937** [comment]
"最后再利用 Nydus Nydus 项目等技术" 多了一个Nydus？

**Reply by isno** [reply]
谢谢，已修改

**Comment by zjyyzhengli** [comment]
“图 7-10 Nydus 是怎么工作的 图片来源” -- 这里的图片来源对应的link现在是无效的page，试了下去掉zh的link可以正常显示：https://d7y.io/blog/2022/06/06/evolution-of-nydus/
